packages/controllers/src/snaps/SnapController.ts

```diff
@@ -329,7 +342,7 @@ type SnapControllerArgs = {
 
 type AddSnapBase = {
   id: SnapId;
-  version?: string;
+  versionRange?: string;
```

Comment:
Is there a reason for the rename? Do you think just using `version` is confusing? 🤔

Reply:
Variables in SnapController named `version` are used to handle a *concrete* version of a snap. Range on the other hand, can handle any SemVer compatible range. This was renamed to better represent the functionality of add() family functions